adafruit_hid/us_keyboard_layout.py

```diff
@@ -0,0 +1,218 @@
+# The MIT License (MIT)
```

Comment:
That makes sense to me but its your call. I'm ok either way as long as they are consistent.